docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -38,6 +38,26 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-typedoc"
 		description="Générer des pages Starlight à partir de TypeScript en utilisant TypeDoc."
 	/>
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-blog"
+    title="starlight-blog"
+    description="Ajouter un blogue à votre site de documentation."
+  />
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-openapi"
+    title="starlight-openapi"
+    description="Créer des pages de documentation à partir de spécifications OpenAPI/Swagger."
+  />
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-obsidian"
+    title="starlight-obsidian"
+    description="Publier vos Vaults d'Obsidian sur votre site Starlight."
```

Comment:
```suggestion
    description="Publier vos coffres (vaults) Obsidian sur votre site Starlight."
```

It looks like from a quick Google & Youtube search, the french Obsidian community has settled on "coffre" for vault so we may as well use both so it can be easily found no matter what.

Reply:
Oh okay! Thanks!